docs/docs/developers/guides/js_apps/pay_fees.md

```diff
@@ -107,37 +142,28 @@ const paymentMethod = await SponsoredFeePaymentMethod.new(pxe);
 myAccountManager.deploy({ fee: { paymentMethod } });
 ```
 
-### Pay with FeeJuice
-
-To send a transaction from an already deployed account already holding fee juice:
-(Note: this example is a public token transfer call, but can equally be a private function call)
-
-#include_code pay_fee_juice_send yarn-project/end-to-end/src/e2e_fees/fee_juice_payments.test.ts javascript
-
-**The equivalent to specify fees via CLI...**
-
-import { CLI_Fees } from '/components/snippets';
+## Fee Options
 
-<CLI_Fees />
+Functions pertaining to sending a transaction, such as `deploy` and `send`, each include a `fee` variable defined with the following (optional) parameters:
 
-### Claim and deploy
+#include_code user_fee_options yarn-project/entrypoints/src/interfaces.ts javascript
 
-Here we will use the `claim` object previously from the bridging section, and the corresponding `wallet`, to create the payment method. The payment method is then used to claim fee juice and pay for account contract deployment. Note the function used to bridge fee juice (private/public) should correspond to how the fee juice is claimed.
+### Fee Payment Method
 
-#include_code claim_and_deploy yarn-project/bot/src/factory.ts javascript
+The `paymentMethod` is an object for the type of payment. Each of the implementations can be found [here](https://github.com/AztecProtocol/aztec-packages/blob/#include_aztec_version/yarn-project/aztec.js/src/fee). For example:
 
-#### Claim and Pay
+#include_code fee_juice_method yarn-project/aztec.js/src/fee/fee_juice_payment_method.ts javascript
 
-Calling a function, in this case checking the balance of the fee juice contract:
+### Gas Settings
```

Comment:
Moved the Gas settings to the bottom of the file, since they seem less critical than the rest of the how-to,